SameName.java

```diff
@@ -0,0 +1,4 @@
+package package1;
+
+public class SameName {

```

Comment:
As far as I remember, there was a requirement to create 2 packages and within these packages create a class via the same name. I assume that you are aware of the idea - it's not allowed to create more than one class with the same name in the same package, that's why I can see only one class. Otherwise, please follow the description and create 2 packages and add one class with the same name into each package to check it.